engine-tests/src/test/java/org/terasology/persistence/typeHandling/gson/GsonTypeHandlerLibraryAdapterFactoryTest.java

```diff
@@ -83,9 +83,14 @@ public boolean equals(Object o) {
             }
             TestClass testClass = (TestClass) o;
             return i == testClass.i
-                    && Objects.equals(color, testClass.color)
-                    && Objects.equals(vector4fs, testClass.vector4fs)
-                    && Objects.equals(rectangleiMap, testClass.rectangleiMap);
+                    && java.util.Objects.equals(color, testClass.color)
+                    && java.util.Objects.equals(vector4fs, testClass.vector4fs)
+                    && java.util.Objects.equals(rectangleiMap, testClass.rectangleiMap);
```

Comment:
Any reason why you're using `java.util.Objects.equals` here instead of `Objects.equal(...)` from Guava?